.travis.yml

```diff
@@ -5,6 +5,8 @@ services:
 
 language: java
 before_install:
+  - docker run --name dbfitdb2 -d -p 50000:50000 -v "`pwd`/dbfit-java/db2/src/integration-test/resources:/var/dbfit/dbfit-java/db2/src/integration-test/resources" -e DB2INST1_PASSWORD=db2inst1-pwd -e LICENSE=accept ibmcom/db2express-c:latest db2start
```

Comment:
I think putting these long command lines into shell scripts is probably the way to go. I could call a shell script instead in that `before_install` section.

But I'm not sure where the files from the Git repo are placed on the build server filesystem. Do you know?